sdk/storage/storage-file-datalake/samples/v12/typescript/src/dataLakeServiceClient.ts

```diff
@@ -1,15 +1,11 @@
-// Copyright (c) Microsoft Corporation.
+// Copyright (c) Microsoft corporation.
```

Comment:
Should we capitalize `corporation` ?

Reply:
Good catch, thank you. I just mistyped this.